files/zh-cn/web/javascript/reference/global_objects/string/index.md

```diff
@@ -152,59 +152,22 @@ console.log(eval(s2.valueOf())); // returns the number 4
-| `\"`                                                                                                                                       | 双引号（U+0022 QUOTATION MARK）                                                                      |
-| `\\`                                                                                                                                       | 反斜杠（U+005C REVERSE SOLIDUS）                                                                     |
-| `\n`                                                                                                                                       | 换行符（U+000A LINE FEED; LF）                                                                       |
-| `\r`                                                                                                                                       | 回车符（U+000D CARRIAGE RETURN; CR）                                                                 |
-| `\v`                                                                                                                                       | 垂直制表符（U+000B LINE TABULATION）                                                                 |
-| `\t`                                                                                                                                       | 制表符（U+0009 CHARACTER TABULATION）                                                                |
-| `\b`                                                                                                                                       | 退格键（U+0008 BACKSPACE）                                                                           |
-| `\f`                                                                                                                                       | 换页符（U+000C FORM FEED）                                                                           |
-| `\uXXXX` …其中 `XXXX` 恰好是 `0000`–`FFFF` 范围内的 4 个十六进制；例如，`\u000A` 与 `\n`（换行）相同；`\u0021` 是 `!`                      | `U+0000` 和 `U+FFFF` 之间的 Unicode 码位（Unicode 基于多平台语言）                                   |
-| `\u{X}`…`\u{XXXXXX}` …其中 `X`…`XXXXXX` 是 `0`–`10FFFF` 范围内的 1-6 个十六进制数字；例如，`\u{A}` 与 `\n`（换行符）相同； `\u{21}` 是 `!` | `U+0000` 和 `U+10FFFF` 之间的 Unicode 码位（整个 Unicode）                                           |
-| `\xXX` …其中 `XX` 恰好是 `00`–`FF` 范围内的 2 个十六进制数字；例如，`\x0A` 与 `\n`（换行符）相同；`\x21` 是 `!`                            | `U+0000` 和 `U+00FF` 之间的 Unicode 码位 （Basic Latin 和 Latin-1 Supplement 块；相当于 ISO-8859-1） |
+然而，整个 Unicode 字符集比 65536 大得多。额外的字符以*代理对*（surrogate pairs）的形式存储在 UTF-16 中，代理对是一对 16 位码元，表示一个单个字符。为了避免起义，配对的两个部分必须介于 `0xD800` 和 `0xDFFF` 之间，并且这些码元不用于编码单码元字符。（更准确地说，前导代理的值在 `0xD800` 和 `0xDBFF` 之间（含），而后尾代理的值在 `0xDC00` 和 `0xDFFF` 之间（含）。）每个 Unicode 字符由一个或者两个 UTF-16 码元组成，也称为 _Unicode 码位_（code point）。每个 Unicode 码位都可以使用 `\u{xxxxxx}` 写成一个字符串，其中 `xxxxxx` 表示 1–6 个十六进制数字。
 
-### 长字符串文字
+“孤项代理”是指满足以下描述之一的 16 位码元：
```

Comment:
is there any reference about the  translation of "lone surrogate".

Reply:
I only got "单项代理"/“单独代理” in some blogs. And GPT gives me "孤立代理项" as a reference.
"lone surrogate" means a code unit of surrogate pairs - maybe the high surrogate or the low - show out in a character sequence, and because of the lack of another one, it can't represent the actual context. So I think "孤" is better than "单" to emphasize or imply the lack of another code unit. And a four-character word can be consistent with "高位/低位代理", which is also more catchy.